### PR TITLE
i18n(zh-cn): translate guides/integrations-guide/partytown.mdx

### PR DESCRIPTION
**Pull request by yiyuan9**
#### What kind of changes does this PR include?

- Translated content

#### Description

- translate guides/integrations-guide/partytown.mdx

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+type: integration
+title: '@astrojs/partytown'
+description: 了解如何在你的 Astro 项目中使用 @astrojs/partytown 集成。
+githubURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/partytown/'
+hasREADME: true
+category: other
+i18nReady: true
+---
+
+import Video from '~/components/Video.astro';
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:11]:
```suggestion
```

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+**[Astro 集成][astro-integration]** 允许你在Astro项目中启用 [Partytown](https://partytown.builder.io/)。
+
+## 为什么是 Astro Partytown
+
+Partytown 是一个延迟加载的库，可以帮助将资源密集型脚本转移到 [web worker](https://developer.mozilla.org/zh-cn/docs/Web/API/Web_Workers_API) 中，并且从 [主线程](https://developer.mozilla.org/zh-cn/docs/Glossary/Main_thread) 中移除。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:17]:
```suggestion
Partytown 是一个延迟加载的库，可以帮助将资源密集型脚本转移到 [web worker](https://developer.mozilla.org/zh-CN/docs/Web/API/Web_Workers_API) 中，并且从 [主线程](https://developer.mozilla.org/zh-CN/docs/Glossary/Main_thread) 中移除。
```

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+description: 了解如何在你的 Astro 项目中使用 @astrojs/partytown 集成。
+githubURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/partytown/'
+hasREADME: true
+category: other
+i18nReady: true
+---
+
+import Video from '~/components/Video.astro';
+
+**[Astro 集成][astro-integration]** 允许你在Astro项目中启用 [Partytown](https://partytown.builder.io/)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:13]:
```suggestion
此 **[Astro 集成][astro-integration]** 允许你在Astro项目中启用 [Partytown](https://partytown.builder.io/)。
```

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+```sh
+# 使用 NPM
+npx astro add partytown
+# 使用 Yarn
+yarn astro add partytown
+# 使用 PNPM
+pnpm astro add partytown
+```
+
+如果你遇到任何问题，[请随时在GitHub上向我们报告](https://github.com/withastro/astro/issues)并尝试下面的手动安装步骤。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:38]:
```suggestion
如果你遇到任何问题，[请随时在 GitHub 上向我们报告](https://github.com/withastro/astro/issues)并尝试下面的手动安装步骤。
```

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
++ <script type="text/partytown" src="fancy-analytics.js"></script>  
+
+```
+
+如果你打开 [浏览器开发者工具](https://developer.chrome.com/docs/devtools/open/) 的“网络 Network”标签，你应该可以看到 `partytown` 代理拦截了这个请求。
+
+## 配置
+
+要配置此集成，请将一个 'config' 对象传递给 astro.config.mjs 中的 partytown() 函数调用。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:77]:
```suggestion
要配置此集成，请将一个 'config' 对象传递给 `astro.config.mjs` 中的 `partytown()` 函数调用。
```

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+```js
+// astro.config.mjs
+// ...
+export default defineConfig({
+  integrations: [
+    partytown({
+      config: {
+        //options go here
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:86]:
```suggestion
        // 配置项在这
```

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+      },
+    }),
+  ],
+});
+```
+
+这体现了 [Partytown 配置对象](https://partytown.builder.io/configuration)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:93]:
```suggestion
这与 [Partytown 配置对象](https://partytown.builder.io/configuration) 相对应。
```

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+
+### config.debug
+
+Partytown 自带一个 debug 模式；通过向 config.debug 传入 true 或 false 来启用或禁用它。如果启用了[`debug` 模式](https://partytown.builder.io/debugging)，它会向浏览器控制台输出详细的日志。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:97]:
```suggestion
Partytown 自带一个 `debug` 模式；通过向 `config.debug` 传入 `true` 或 `false` 来启用或禁用它。如果启用了[`debug` 模式](https://partytown.builder.io/debugging)，它会向浏览器控制台输出详细的日志。
```

File: src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+
+如果不设置此选项，在[dev](/zh-cn/reference/cli-reference/#astro-dev) 或 [preview](/zh-cn/reference/cli-reference/#astro-preview) 模式下,`debug` 模式默认启用。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/partytown.mdx:99]:
```suggestion
如果不设置此选项，在 [dev](/zh-cn/reference/cli-reference/#astro-dev) 或 [preview](/zh-cn/reference/cli-reference/#astro-preview) 模式下，`debug` 模式默认启用。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**